ietf/name/fixtures/names.json

```diff
@@ -6025,7 +6025,7 @@
   },
   {
     "fields": {
-      "desc": "The internet drafts ticketing system",
+      "desc": "The Internet-Drafts ticketing system",
```

Comment:
These will need to be changed in the production DB, either by migration or manually

Reply:
Good catch. This is small enough to do by hand using the admin I think.

Reply:
I made these adjustments in production using the admin